package.json

```diff
@@ -11,7 +11,10 @@
     "htmlhint": "htmlhint \"src/**/*.html\"",
     "tsc": "tsc",
     "install-hooks": "git config core.hooksPath .hooks",
-    "test": "jest"
+    "test": "jest",
+    "lint:eslint": "npm run eslint -- --fix",
+    "lint:stylelint": "npm run stylelint -- --fix",
+    "lint:all": "npm run lint:eslint && npm run lint:stylelint"
```

Comment:
Can you rename this to `fixlints:` and rename `lint:all` to `fixlints`

Reply:
sure 👍